test/fixtures/bundler/Gemfile

```diff
@@ -26,8 +26,7 @@ end
 gem "pathed-gem-fixture", path: "pathed-gem-fixture"
 
 # verify https://github.com/github/licensed/issues/71
-git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
-gem "mini_racer", "0.3.1"
+gem "loofah", "2.18.0"
```

Comment:
can this use nokogiri directly or do you know if having it as an indirect dependency makes a difference?  it's been awhile since that issue was opened and then fixed, but I think mini_racer was used here instead of libv8 because it matched the repro steps of the issue.

if we need to break away from libv8, maybe we can also get away from an indirect dependency?

Reply:
oh also, can you add a comment so that it's clear whats being tested since the repro doesn't match the issue link any longer?  feel free to ignore my suggested text and use your own if you would prefer 👍 

```suggestion
# depends on platform-dependent nokogiri installation
gem "loofah", "2.18.0"
```